tools/profile_example.txt

```diff
@@ -161,6 +161,35 @@ but we're lacking the symbol translation. This is a common for all profilers
 on Linux, and is usually fixable. See the DEBUGGING section of the profile(8)
 man page.
 
+You can also profile different process:
+# ./profile -p 2040 1316151
```

Comment:
should the option look like `-p 2040 -p 1316151`? What is the syntax `perf` allows multiple pid/tid's?

Reply:
perf just like: -p 2040,1316151 
should change to it?

Reply:
Yes, it would be good to following perf conventions.

Reply:
Please also mention perf convention in your commit message.

Reply:
resolved, pls review it